### PR TITLE
Adding some documentation for how to create a custom base image

### PR DESCRIPTION
**Pull request by htrivedi99**

### CODE REVIEW

File: docs/guides/base-images.mdx [hunk 1]
```diff
@@ -7,6 +7,64 @@ description: "A guide to configuring a base image for your truss"
 Model serving enviroments will often be standardized as container images to avoid wrangling python, system, and other requirements needed to run your model on every deploy.
 Leverage your existing container artifacts by bringing your own base image to Truss.
 
+While the default image for the truss server in adequate for most use cases, there may come a time when you require a custom base image for your truss.
+For example, maybe the python packages required for you project are not compatible with the ones installed by default on Truss.
+In a situation like this, you can create your own base image based on the default truss image.
+
+## Creating a custom base image
```
Comment (squidarth) [inline docs/guides/base-images.mdx:14]: what do you think about having the `Setting a base image in config.yaml` section first, and then having this after that, with a transition like "If you don't have a base image ready yourself, and need to make one..."

**Review by squidarth — APPROVED** [review]
this is awesome, really appreciate you taking the time to help improve our docs!

Tagging our technical writer @philipkiely-baseten in case he has any feedback

File: docs/guides/base-images.mdx (hunk 1 above)
Comment (htrivedi99) [reply docs/guides/base-images.mdx:14]: yeah, I think that format works as well

**Review by philipkiely-baseten — APPROVED** [review]
Hey @htrivedi99 sorry I missed this!

Overall content looks good, want to get a couple style guide things in though:

* Capitalize Truss, Docker, etc
* Be consistent on H2 and H3 tags for what is a section vs subsection
* Be consistent on -ing vs regular verbs on H2 and H3 tags (standardize on -ing verbs)
* Don't put H3/any heading tags inside of <Tip> or similar

Otherwise, this looks good! Thank you for contributing some docs!

**Pull request merged**